# Spatial relationship between two scene objects

int32  id1                                # First scene object's ID
std_msgs/String spatial_relationship      # Spatial relationship between two scene objects
int32  id2                                # Second scene object's ID